source/module_hamilt_general/module_xc/xc_functional_vxc.cpp

```diff
@@ -31,6 +33,7 @@ std::tuple<double,double,ModuleBase::matrix> XC_Functional::v_xc(
     double vtxc = 0.0;
     ModuleBase::matrix v(GlobalV::NSPIN, nrxx);
 
+    ModuleBase::Memory::record("xc_functional_vxc::matrix",sizeof(double)*GlobalV::NSPIN*nrxx);
```

Comment:
In this function, the matrix `v` is the return variable, will be moved to the caller, please record it when it is called.